WISE-Community/WISE
# Fixed a bug where the grade by student page does not load after a step is deleted. Also fixed a problem with old annotations persisting if the step is deleted and then recreated and has the same node id

How to test
1. Create a run.
2. Log in as a student and do some work on a step.
3. Log in as the teacher and give the student a score on the work.
4. Delete the step.
5. Go to the Grading Tool and open the Grade By Student page. Make sure the page loads.

How to test part 2
6. Go back to the Authoring Tool and add a step back. Make sure it has the same node id as the step you deleted earlier. Add a new open response component to the step.
7. Go to the Grading Tool and open the Grade By Student page. Make sure the previous score you gave does not show up again.

Closes #1514
